enterprise/backend/test/metabase_enterprise/serialization/test_util.clj

```diff
@@ -126,14 +126,14 @@
           (when (.exists (io/file dump-dir))
             (.delete (io/file dump-dir))))))))
 
-(defmacro with-random-dump-dir {:style/indent 2} [[dump-dir-binding prefix] & body]
+(defmacro with-random-dump-dir {:style/indent 1} [[dump-dir-binding prefix] & body]
   `(do-with-random-dump-dir ~prefix (fn [~dump-dir-binding] ~@body)))
 
 (defmacro with-world
   "Run test in the context of a minimal Metabase instance connected to our test database."
+  {:style/indent 0}
   [& body]
-  `(with-temp-dpc [Database   [{~'db-id :id} (into {:name temp-db-name} (-> (data/id)
-                                                                            Database
+  `(with-temp-dpc [Database   [{~'db-id :id} (into {:name temp-db-name} (-> (data/db)
```

Comment:
You cannot invoke a model in Toucan 2, so I replaced this with an equivalent form.